src/content/enterprise/aks.mdx

```diff
@@ -101,29 +101,12 @@ Want to use Okteto with a bigger team? [Let's talk](https://okteto.com/#talktous
 
 ## Installing Okteto
 
-### Creating the Okteto namespace
+### Create the Okteto Namespace
 
-Create a dedicated namespace for your Okteto instance:
+Run the command below to create a namespace for okteto:
 
 ```console
-$ kubectl apply -f https://charts.okteto.com/namespace.yaml
-```
-
-*The `namespace.yaml` manifest assumes you'll be installing the application into the `okteto` namespace. If you are not doing this, you'll need to update the file before creating the namespace.*
-
-Jump to the next step if you are [bringing your own certificates](/docs/enterprise/administration/certificates/).
-
-If you are already running Cert Manager in your cluster, add this setting to your Okteto helm configuration:
-
-```
-cert-manager:
-  enabled: false
-```
-
-Otherwise, if you want to install Cert Manager as part of the Okteto installation, run the command below:
-
-```console
-$ kubectl apply -f https://charts.okteto.com/crds.yaml
+$ kubectl create namespace okteto
```

Comment:
We could, but it requires a greater refactor to the installation guides because we are creating secrets on the okteto namespace before running `helm install`. In order to reduce moving parts, I would do it in a different iteration

Reply:
ok